### DISCUSSION TITLE
為何最近的動漫常常讓主角死掉

### DISCUSSION THREAD

**Discussion by Terryyyyyyyyyyyy** [1101-GS4719-C/Python-Community-GS4719-C#1776] [General]
以前，以比較長期連載的漫畫來說，通常不會讓主角或重要人物輕易的死亡，甚至會賦予他超越其他人的強度。
但現在越來越多的漫畫傾向送主角或重要角色下去，如：咒術迴戰的五條悟、進擊的巨人的艾連及團長、火影忍者的自來也等。
我認為可能跟兩點有關：
1.以主角或重要人物為主的漫畫已經失去熱度了，反而適當的送主角或重要人物下去更能獲得熱度，更有更多的可能性。
2.以主角為王的漫畫已經太多了，只好透過送主角下去來突出自己的漫畫。

**Comment by Cerosop** [comment]
我好像又被劇透了欸，雖然這都是些熱門動漫但還是希望可以不要提到確切人名

**Comment by Chengbosco** [comment]
送主角或重要人物下去是沒差啦，但你要邏輯上說服觀眾，不然就是感情。向艾o文的撥手和艾o的砍頭都很好啊，阿五o悟是什麼鬼。
主要可以分成兩個，角色發展： 主角的死亡可能是為了推動故事中其他角色的發展。其他角色在主角的離去中可能會經歷成長，學習新的技能或者面對新的挑戰。打破常規： 選擇讓主角死掉可能是為了打破故事的常規，避免故事變得太過預測性。這樣的轉變可以為觀眾提供新的視角和體驗。
![下載 (2)](https://github.com/NCU-GS4719-Python/Python-Community/assets/144880105/c3f3045f-a478-48b7-81cc-20018079b7b3)

**Comment by lazywulf** [comment]
送主角領便當絕對沒問題，但你不能毫無邏輯就送人下去，然後擺個自認有趣就蒙混過關。
就不說是哪部，還辯說自己少畫一話。明明可以好端端寫完故事(我個人覺得到那段已經不大優了，整個戰鬥感覺已經很沒感覺了)，你讓一個到處開掛(不是自己的能力、滅火器)的反派，最後關頭來個「哎呀我還會這招喔」，然後反派斬完人也草草講個幾句就結束了。
哪個腦袋有問題的作者編的啊？要送下去也給我隆重送下去啊，太隨便了吧。
你就算後面把人送回來我也不覺得會多好看了。
